nougat/evaluete.py

```diff
@@ -349,3 +349,84 @@ def computeGC(sequence):
     gcFraction = float(gcCount) / totalBaseCount
     return gcFraction
 
+
+def computeAssemblyStats(sample_config):
+
+    outfile = os.path.join("contig_stats", "contiguity.out")
+    if not os.path.exists("contig_stats"):
+        os.makedirs("contig_stats")
+
+    try:
+        sequence = sample_config["reference"]
+        minlength = sample_config["minCtgLength"]
+        genomesize = sample_config["genomeSize"]
+    except KeyError, e:
+        if e is "minCtgLength":
+            minlength = 1000
```

Comment:
you could have used `sample_config.get('minCtgLength','1000')` above in `try` block, so avoided this `if` block ? (Nothing wrong, just curious :wink: )